browser/directives/pathValidator.js

```diff
@@ -44,11 +44,12 @@ function pathValidator() {
       mCtrl.$validators['notSelected'] = isSelected;
       mCtrl.$validators['notAbsolute'] = isAbsolute;
       mCtrl.$validators['tooLong'] = validateLength;
-      mCtrl.$validators['hasSpaces'] = hasNoSpaces;
       if(Platform.OS == 'win32') {
         mCtrl.$validators['invalidAscii'] = validateAscii;
         mCtrl.$validators['invalidFormat'] = validateFormatWindows;
         mCtrl.$validators['invalidDisk'] = validateDisk;
+      } else {
+        mCtrl.$validators['hasSpaces'] = hasNoSpaces;
```

Comment:
Any particular reason not to allow spaces on mac?

Reply:
There is no reason to ban locations with spaces on macOS.